src/helpers/MiscFunctions.cpp

```diff
@@ -722,6 +722,22 @@ int64_t configStringToInt(const std::string& VALUE) {
     return std::stoll(VALUE);
 }
 
+Vector2D configStringToVector2D(const std::string& VALUE) {
+    size_t      spacePos = VALUE.find(' ');
```

Comment:
this function should not accept anything but `123 456` or `123, 456`

Reply:
wouldn't allowing commas break the window rule syntax

Reply:
thats why allow both

Reply:
no like, say I have this rule:

`windowrulev2 = maxsize 500 500, class:foot`

this works, but this rule:

`windowrulev2 = maxsize 500, 500, class:foot`

now the second `500` is interpreted as the window var

Reply:
thats why there they would be parsed with a space

Reply:
ok check now